presto-docs/src/main/sphinx/release/release-0.219.rst

```diff
@@ -0,0 +1,62 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+
+* Fix a correctness bug for queries with a ``LIMIT`` on top of a ``FULL OUTER JOIN``.
+* Fix a correctness bug in how word boundaries are handled for Joni regular expression functions.
+* Fix a correctness bug that would occur if two tables with mismatching but compatible bucket counts are joined whiles filters are applied to one of the tables.
+* Fix Parquet predicate pushdown for ``smallint`` and ``tinyint`` types.
```

Comment:
what didn't work about it before?

Reply:
@nezihyigitbasi

Reply:
It was previously causing a query to fail with an `IllegalArgumentException`. Maybe we can say 

```
Fix Parquet predicate pushdown for ``smallint`` and ``tinyint`` types, which was previously causing the query to fail.
```

Reply:
I think the `which` won't be necessary in this case.

Reply:
Fix query failures caused by Parquet predicate pushdown for ``smallint`` and ``tinyint`` types.